internal/testutils/db.go

```diff
@@ -261,8 +261,13 @@ func loadDumpIntoDB(t *testing.T, dumpPath, dbPath string) {
 			var s string
 			// Looping through the columns to ensure that the values will be ordered as supposed to.
 			for i, col := range table.Cols {
-				values[i] = row[col]
-				if col == "last_online_auth" && values[i] == "RECENT_TIME" {
+				if col == "last_online_auth" {
+					values[i] = ParseTimeWildcard(row[col]).Unix()
+				} else {
+					values[i] = row[col]
```

Comment:
I would put is before the if, then override if it’s last_online_auth and thus, removing the else clause.

Reply:
I've also realized that I forgot to remove the previous code there as well 🤡